Tutorials/whatiswot/2-Payloads_and_Headers/2-Payloads_and_Headers.txt

```diff
@@ -23,7 +23,7 @@ Let's continue with the parcel truck example. In that example, the header would
 
 This is what an HTTP header can look like. A header can contain information like request context, response context, status and it can indicate payload type.
 
-There are different payload types such as  JSON, XML, CBOR, Text, Audio and Video and more.
+There are different payload types such as JSON, XML, CBOR , Text, Audio and Video and more. 
```

Comment:
Many odd space characters added by PR #164. This line has two -- one midline (`CBOR ,`), one before EOL (`more. `). Some lines now have 2 or 3 space chars before EOL. These errors make me wonder whether (at least some of) this PR represents regression of previously merged PR...?

Reply:
@TallTed thanks for the review. These are exported from our voice generator tool (murf.ai) and I think the regression happened on their side :/